templates/ci/docker.yml

```diff
@@ -0,0 +1,63 @@
+# Docker
+# Build, tag, push, or run Docker images, or run a Docker command.
+# More Information on https://go.microsoft.com/fwlink/?linkid=848006
+# CodeLaunch YAML help: https://aka.ms/yaml
+
+queue:
+  name: Hosted VS2017                                       # Hosted Linux Preview, Default
+
+variables:
+#  system.debug: true
+
+  dockerRegistryConnectionEndpoint: ''                      # The unique name or the GUID Id of VSTS service connection endpoint
```

Comment:
does this work if you specify a value?

Reply:
As far I remember (too many options tried :-)) yes, it works.  I'll double-check. Thanks.

Reply:
As far I remember (too many options tried :-) ) yes, it works.  I'll double-check. Thanks.